src/nomad_simulations/schema_packages/utils/utils.py

```diff
@@ -168,3 +170,54 @@ def get_composition(children_names: 'list[str]') -> str:
     children_count_tup = np.unique(children_names, return_counts=True)
     formula = ''.join([f'{name}({count})' for name, count in zip(*children_count_tup)])
     return formula if formula else None
+
+
+def check_simulation_cell(cell_1: 'Cell', cell_2: 'Cell') -> bool:
+    """
+    Check if the two `Cell` objects are the same by checking if the defined `positions` are all matching. If
+    the objects are `AtomicCell`, it checks if the `AtomsState[*].chemical_symbol` are the same.
+
+    Args:
+        cell_1 (Cell): The first `Cell` to compare.
+        cell_2 (Cell): The second `Cell` to compare.
+
+    Returns:
+        bool: True if the cells are the same, False otherwise.
+    """
+    # If any of the cells is None or the positions are empty, return False
+    if cell_1 is None or cell_2 is None:
+        return False
+    if cell_1.positions is None or cell_2.positions is None:
+        return False
+
+    # The `positions` should have the same length (same number of positions)
+    if len(cell_1.positions) != len(cell_2.positions):
+        return False
+    n_positions = len(cell_1.positions)
+
+    # Check that all the `positions`` of `cell_1` match with the ones in `cell_2`
+    check_positions = []
+    for i1, pos1 in enumerate(cell_1.positions):
+        for i2, pos2 in enumerate(cell_2.positions):
+            if np.allclose(pos1, pos2):
```

Comment:
are you not supposed to return False here is they are not close?

Reply:
I am checking one position of the `cell_1` with respect to all the other positions in `cell_2`, so no, I can only continue.

This implementation I did is not optimized at all... I would work on it, but first, I'll check ase for their implementation.